dnf-behave-tests/dnf/downgrade.feature

```diff
@@ -87,96 +85,44 @@ Scenario: Downgrade a package that was installed via rpm
         | Replaced  | flac-0:1.3.3-3.fc29.x86_64 | External User | @System               |
 
 
-# @dnf5
-# TODO(nsella) different exit code 0
-# TODO(nsella) different stderr
+# https://github.com/rpm-software-management/dnf5/issues/524
 Scenario Outline: Check <command> exit code - package does not exist
   Given I use repository "dnf-ci-fedora"
    When I execute dnf with args "<command> non-existent-package"
    Then the exit code is 1
-    And stderr is
-    """
-    Error: No packages marked for <command>.
-    """
+    And stderr contains "No match for argument: non-existent-package"
 
 Examples:
     | command   |
     | upgrade   |
     | downgrade |
 
 
-# @dnf5
-# TODO(nsella) different exit code 0
-# TODO(nsella) different stdout
-# TODO(nsella) different stderr
-Scenario: Check downgrade exit code - package not installed
+Scenario Outline: Check <command> exit code - package not installed
   Given I use repository "dnf-ci-fedora"
-   When I execute dnf with args "downgrade flac"
+   When I execute dnf with args "<command> flac"
    Then the exit code is 1
-    And stdout is
-    """
-    <REPOSYNC>
-    Packages for argument flac available, but not installed.
-    """
-    And stderr is
-    """
-    Error: No packages marked for downgrade.
-    """
-
-
-# @dnf5
-# TODO(nsella) different exit code 0
-# TODO(nsella) different stdout
-# TODO(nsella) different stderr
-Scenario: Check upgrade exit code - package not installed
-  Given I use repository "dnf-ci-fedora"
-   When I execute dnf with args "upgrade flac"
-   Then the exit code is 1
-    And stdout is
-    """
-    <REPOSYNC>
-    No match for argument: flac
-    """
-    And stderr is
-    """
-    Package flac available, but not installed.
-    Error: No packages marked for upgrade.
-    """
-
-
-# @dnf5
-# TODO(nsella) different stdout
+    And stderr contains "Packages for argument 'flac' available, but not installed."
+
+Examples:
+    | command   |
+    | upgrade   |
+    | downgrade |
+
+
 @bz1759847
 Scenario: Check upgrade exit code - package already on the highest version
   Given I use repository "dnf-ci-fedora"
     And I successfully execute dnf with args "install flac-0:1.3.3-3.fc29.x86_64"
    When I execute dnf with args "upgrade flac"
-   Then the exit code is 0
-    And stdout is
-    """
-    <REPOSYNC>
-    Dependencies resolved.
-    Nothing to do.
-    Complete!
-    """
+   Then the exit code is 1
```

Comment:
dnf5 has the same behavior as dnf4 so during an upgrade if the package is already at the highest version the exit code is 0.